File: Java/MathematicalPrograms/SimpleInterest.java
import java.util.Scanner;
class SimpleInterest {
    public static void main(String[] args) {
        Scanner myObj = new Scanner(System.in);
        System.out.println("Enter the Principal Amount");
        int principal = myObj.nextInt();
        System.out.println("Enter the Time in Years");
        int time = myObj.nextInt();
        System.out.println("Enter the Rate of Interest");
        int rateofinterest = myObj.nextInt();
        int simpleinterest = principal*time*rateofinterest;
        int amount = principal + simpleinterest;
        System.out.println("Simple Interest is " + simpleinterest + " Rupees. Total Amount is " + amount + " Rupees");
    }
}

File: C++/MathematicalPrograms/SimpleInterest.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
	int principal, time, rateofinterest;
	cout << "Enter the Principal Amount";
	cin >> principal;
	cout << "Enter the Time in Years";
	cin >> time;
	cout << "Enter the Rate of Interest";
	cin >> rateofinterest;
	int simpleinterest = principal*time*rateofinterest;
	int amount = simpleinterest+principal;
	cout << "Simple Interest is " << simpleinterest << " Rupees. Total Amount is " << amount << " Rupees";
}
	